build_tools/benchmarks/benchmark_helper.py

```diff
@@ -141,12 +146,61 @@ def _dump_cmds_handler(e2e_test_artifacts_dir: pathlib.Path,
   print(*lines, sep="\n")
 
 
+def merge_results(benchmark_results_files: List[io.TextIOWrapper]):
+  # Merges the benchmark results from `right` into `left` and returns the updated `left`
+  def _merge_two_resultsets(left, right):
+    # Parses a JSON benchmark results file and makes some sanity checks
+    def _load(file):
+      try:
+        content = json.load(file)
+      except json.JSONDecodeError as e:
+        raise RuntimeError('"{}" seems not to be a valid JSON file: {}'.format(
+            file.name, e.msg))
+      if not isinstance(content, dict):
+        raise RuntimeError(
+            '"{}" seems not to be a valid benchmark-results-file (No JSON struct as root element).'
+            .format(file.name))
+      if 'commit' not in content:
```

Comment:
IREE usually use double quote when possible

Reply:
I was expecting the code formatting to take care of that. But anyway - changed.